File: Feitos em Python 3/2533.py
while True:
    try:
        times=int(input())
        dom=0.0
        dem=0.0
        i=0
        while i<times:
            aux=input().split()
            n=int(aux[0])
            c=int(aux[1])
            dom += (n*c)
            dem += (c*100)
            i+=1
        resu =0.0
        resu += (dom/dem)
        print("{:.4f}".format(resu))
    except EOFError:
        break

File: Feitos em C/2533.c
#include <stdio.h>
int main() {
	int times;
	while(scanf("%d",&times)==1){
		double dom=0,dem=0;
		int i;
		for(i=0;i<times;i++){
			int n,c;
			scanf("%d %d",&n,&c);
			dom+= (n*c);
			dem+= (c*100);
		}
		double resu = dom/dem;
		
		printf("%.4f\n",resu);
	}
    return 0;
}
